helm/cluster-eks/ci/ci-values.yaml

```diff
@@ -1,14 +1,16 @@
-metadata:
-  organization: "test"
-baseDomain: example.com
+global:
+  metadata:
+    organization: "test"
+    description: "test description"
+  baseDomain: example.com
```

Comment:
Needs to be moved to connectivity
global.connectivity.baseDomain

Reply:
done